### PR TITLE
prometheus-vmware-rules: onboard to sharded concept

### PR DESCRIPTION
**Pull request by richardtief**
This is a temporary templating to accompany the progress as we roll out region by region.

### CODE REVIEW

File: prometheus-rules/prometheus-vmware-rules/values.yaml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # The name of the Prometheus to which the alert- and aggregation rules will be assigned to.
 prometheusName: vmware
+shardedRegions:
+  - qa-de-1
```
Comment (viennaa) [inline prometheus-rules/prometheus-vmware-rules/values.yaml:4]: Shouldn't we default to non-sharded? Wouldn't bake in a static region here.

Comment (richardtief) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:4]: This is our region list that we maintain as long as we roll out. After that this entry goes away again.

Comment (richardtief) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:4]: In other words, we need a switch somewhere.

Comment (richardtief) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:4]: And the default is non-sharded. This `else` condition
https://github.com/sapcc/helm-charts/blob/f3ca66bb5a61ffa74e21a3fff1ddeee0ad985ea2/prometheus-rules/prometheus-vmware-rules/templates/prometheus-aggregations.yaml#L19-L34

Comment (viennaa) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:4]: Can you please add a comment for this variable describing this? Something like:  `static list to check if region is a sharded region, to be removed later`

**Review by viennaa — APPROVED** [review]

**Pull request merged**